### DISCUSSION TITLE
markdown-preview TOC links not working

### DISCUSSION THREAD

**Discussion by stefaniecg** [pulsar-edit/.github#131] [Pulsar Support]
I am using the package [markdown-preview 0.160.2](https://web.pulsar-edit.dev/packages/markdown-preview) to create a preview of my `.md` files. 

But, the table of content (TOC) links do not work, i.e. they do not jump to the desired part of the preview. 

Could this be fixed?

```
Title of markdown file
========================

**TABLE OF CONTENT**:
<!-- TOC depthFrom:1 depthTo:6 withLinks:1 updateOnSave:1 orderedList:0 -->

- [subtittle1](#subtittle1)
- [subtitle2](#subtitle2)

<!-- /TOC -->

# subtittle1

# subtitle2
```

**Comment by Tunkert** [comment]
Wouldn't you just have each section as ## subtitle 1 ex with a space between the two hashtags for a second level heading. It looks like it is formatting it correctly on my end.

I just started a new YT channel where I'm covering the Pulsar editor in a series. I could put a video up on it if you'd be interested.

**Reply by stefaniecg** [reply]
@Tunkert , hi thank you for your reply. The problem is that the links in the TOC do not jump to the linked seciton in the markdown-preview.

**Comment by Tunkert** [comment]
Got ya.

**Comment by Daeraxa** [comment]
For what it is worth it doesn't seem like this is working in Atom either.

**Reply by confused-Techie** [reply]
Yeah links have never worked in the markdown package.

So this isn't a bug, this would be a new feature request.

And by links I mean those to the same page, to local resources, or to outside webpages.

**Reply by Daeraxa** [reply]
It is a request that makes sense I think though, how odd. Never really used the markdown preview before but that seems like basic functionality.
